.travis.yml

```diff
@@ -249,4 +249,45 @@ jobs:
         - *validate-openapi-v3-json
         - *validate-openapi-v3-yaml
 
+    - php: '7.3'
+      env: APP_ENV=absolute_url
```

Comment:
Still the same problem as https://github.com/api-platform/core/pull/3198#discussion_r336795993

We should not duplicate (parts of) our test suite just for one feature, or require extra CI jobs for it.

Reply:
What do you propose @teohhanhui, I don't see other way of testing this in behat

Reply:
We need to have an attribute in the ResourceMetadata that changes this parameter resource-wise, then add a test on a new resource

Reply:
I agree, using an attribute, and the new "defaults" mechanism looks better!

Reply:
I've started to update yesterday and it is working but there is one issue with this approach that I'm not sure how to tackle. How to update the Entrypoint in every format and the @context for errors in JSON-LD. Those are not tight to resource. Any idea ?

Reply:
would you use the new default mechanism to still set a parameter in the container for those cases ?

Reply:
updated, switched to resource attribute @soyuka  @dunglas 
no more suite duplication @teohhanhui